sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveExternalCatalog.scala

```diff
@@ -787,7 +787,7 @@ private[spark] class HiveExternalCatalog(conf: SparkConf, hadoopConf: Configurat
     val storageWithLocation = {
       val tableLocation = getLocationFromStorageProps(table)
       // We pass None as `newPath` here, to remove the path option in storage properties.
-      updateLocationInStorageProps(table, newPath = None).copy(
+      table.storage.copy(
```

Comment:
In the comments above:
```
    // Internally we store the table location in storage properties with key "path" for data
    // source tables. Here we set the table location to `locationUri` field and filter out the
    // path option in storage properties, to avoid exposing this concept externally.
```
```
// We pass None as `newPath` here, to remove the path option in storage properties.
```
The property `path` is removed intentionally. 
To finish to a `TODO` comment in unit test is not a good reason to keep it.

Reply:
So we should remove the TODO comment/work?